docker/bcl_convert/Dockerfile

```diff
@@ -1,17 +1,17 @@
-FROM continuumio/miniconda3:4.6.14
+FROM --platform=amd64 continuumio/miniconda3:latest
```

Comment:
I know for package libraries we want to specify a version for consistency and try not to use latest but I don't know if we have to do that for miniconda...